src/renderer/components/message/MessageList.tsx

```diff
@@ -149,6 +150,14 @@ export default function MessageList({
   )
 }
 
+/** Object holding type information about a chat for messages in that chat */
+export type ConversationType = {
```

Comment:
Should this type distinguish between device chat, one to one chat, group chat...?

Reply:
The reason for it is rather to provide some information about the chat to the messages without passing all of the `FullChat` type down.
Essentially yes at the moment it only provides information about the chattype.